scripts/script.js

```diff
@@ -0,0 +1,48 @@
+const cocktailApp = {};
+
+cocktailApp.init = () => {
+  cocktailApp.getIngredientName();
+};
+
+// Get user's input and fetch the data from API
+cocktailApp.getIngredientName = () => {
+  
+  const formE = document.querySelector('form');
+  
+  // wasn't sure if I should call this one here or one function above
+  const drinkList = document.querySelector('.drink__list');
+  
+  formE.addEventListener('submit', (e) => {
```

Comment:
For example:
`cocktailApp.form.addEventListener()`
(We could also use the this keyword if the getIngredientName method is switched to function syntax instead of arrow syntax.